src/__integ__/customEventAPI.test.ts

```diff
@@ -0,0 +1,93 @@
+import { Selector } from 'testcafe';
```

Comment:
Issue: Our integration tests start to perform very poorly when there are a lot of tests. Further investigation is needed to determine if the flakiness is from testcafe or RUM's behavior between tests. For now, I've separated the custom events into two files to mitigate the problem.